src/components/common/scheduleTable.jsx

```diff
@@ -1,45 +1,41 @@
-import React from "react";
-import Trash from "./trash";
-import Leaf from "./leaf";
-import Recycle from "./recycle";
-const ScheduleTable = ({ services, renderWeekOfDay }) => {
-  // const {services,renderWeekOfDay} = props;
+import React from 'react';
+import Icon from './Icon';
+const ScheduleTable = ({ services, renderDayofWeek }) => {
 
-   return (
+  return (
     <table className="table table-bordered table-sm">
       <thead>
         <tr>
           <th>Type</th>
-          <th>Image</th>
           <th>Collection Days</th>
           <th>Next Collection</th>
         </tr>
       </thead>
       <tbody>
         {services.map(service => (
           <tr key={service._id}>
-            <td>{service.type}</td>
             <td align="center">
-              {service.type === "Trash" ? (
-                <Trash />
-              ) : service.type === "Leaf" ? (
-                <Leaf />
+              {service.type.toLowerCase() === "trash" ? (
+                <Icon iconClass={service.type.toLowerCase()} />
+              ) : service.type.toLowerCase() === "leaf" ? (
+                <Icon iconClass={service.type.toLowerCase()} />
               ) : (
-                <Recycle />
-              )}
+                    <Icon iconClass={service.type.toLowerCase()} />
+                  )}
+              {service.type}
             </td>
             <td>{service.collectionDays}</td>
             <td>
-              {service.type === "Trash"
-                ? renderWeekOfDay(1)
-                : service.type === "Leaf"
-                ? renderWeekOfDay(3)
-                : renderWeekOfDay(2)}
+              {service.type.toLowerCase() === "trash"
+                ? renderDayofWeek(service.type.toLowerCase())
```

Comment:
since 
```js
renderDayofWeek(service.type.toLowerCase()
```

is the same every time you check what type of service type it is, you don't need the conditionals, you handle the check inside the renderDayOfWeek function.

Reply:
removed